addons/payment/tests/test_payment_transaction.py

```diff
@@ -4,6 +4,8 @@
 
 from odoo.addons.payment.tests.common import PaymentCommon
 
+from odoo.tools import mute_logger
+
```

Comment:
```suggestion
from odoo.tools import mute_logger

from odoo.addons.payment.tests.common import PaymentCommon
```
Import order:
1. Standard Libs
2. Other libs
3. Odoo framework things
4. Odoo modules things

Reply:
Done :)